crates/runtime/src/http/routes.rs

```diff
@@ -18,7 +18,7 @@ pub(crate) fn routes(
     Router::new()
         .route("/health", get(|| async { "ok\n" }))
         .route("/v1/datasets", get(v1::datasets::get))
-        .route("/v1/models/:name/predictions", get(v1::inference::get))
+        .route("/v1/models/predict", post(v1::inference::post))
```

Comment:
@mitchdevenport don't we want to keep both? The option to batch and the option to use the named API?